src/main/java/com/tjtanjin/steve/Steve.java

```diff
@@ -0,0 +1,32 @@
+package com.tjtanjin.steve;
+
+import com.tjtanjin.steve.commands.CommandHandler;
+import com.tjtanjin.steve.parser.Parser;
+import com.tjtanjin.steve.storage.StorageHandler;
+import com.tjtanjin.steve.tasks.TaskHandler;
+
+/**
+ * Steve class creates and allocates all key objects.
+ */
+public class Steve {
+
+    private final Parser parser;
```

Comment:
PARSER?

Reply:
Thank you for pointing this out too, fixed in the latest commit :)